buildenv/jenkins/variables/defaults.yml

```diff
@@ -466,6 +576,63 @@ x86-64_linux_cm:
       <<: *defaultTests
       ? special.system
 #========================================#
+# Linux x86 64bits Compressed Pointers /w JITSERVER
+#========================================#
+x86-64_linux_jit:
+  boot_jdk:
+    8: '/usr/lib/jvm/jdk-7'
+    11: '/usr/lib/jvm/jdk-11'
+    12: '/usr/lib/jvm/jdk-11'
+    13: '/usr/lib/jvm/adoptojdk-java-12'
+    next: '/usr/lib/jvm/adoptojdk-java-13'
+  release:
+    8: 'linux-x86_64-normal-server-release'
+    11: 'linux-x86_64-normal-server-release'
+    12: 'linux-x86_64-server-release'
+    13: 'linux-x86_64-server-release'
+    next: 'linux-x86_64-server-release'
+  freemarker: '/home/jenkins/freemarker.jar'
+  openjdk_reference_repo: '/home/jenkins/openjdk_cache'
+  node_labels:
+    build:
+      8: 'ci.role.build && hw.arch.x86 && sw.os.cent.6'
+      11: 'ci.role.build && hw.arch.x86 && sw.os.cent.6'
+      12: 'ci.role.build && hw.arch.x86 && sw.os.cent.6'
+      13: 'ci.role.build && hw.arch.x86 && sw.os.cent.6'
+      next: 'ci.role.build && hw.arch.x86 && sw.os.cent.6'
+  extra_getsource_options:
+    8: '--openssl-version=1.1.1d'
+    11: '--openssl-version=1.1.1d'
+    12: '--openssl-version=1.1.1d'
+    13: '--openssl-version=1.1.1d'
+    next: '--openssl-version=1.1.1d'
+  extra_configure_options:
+    8: '--enable-cuda --with-cuda=/usr/local/cuda-9.0 --with-openssl=fetched --enable-jitserver=yes'
+    11: '--enable-cuda --with-cuda=/usr/local/cuda-9.0 --with-openssl=fetched --enable-jitserver=yes'
+    12: '--enable-cuda --with-cuda=/usr/local/cuda-9.0 --with-openssl=fetched --enable-jitserver=yes'
+    13: '--enable-cuda --with-cuda=/usr/local/cuda-9.0 --with-openssl=fetched --enable-jitserver=yes'
+    next: '--enable-cuda --with-cuda=/usr/local/cuda-9.0 --with-openssl=fetched --enable-jitserver=yes'
+  build_env:
+    cmd:
+      8: 'source /opt/rh/devtoolset-7/enable'
+      11: 'source /opt/rh/devtoolset-7/enable'
+      12: 'source /opt/rh/devtoolset-7/enable'
+      13: 'source /opt/rh/devtoolset-7/enable'
+      next: 'source /opt/rh/devtoolset-7/enable'
+  excluded_tests:
+    11:
```

Comment:
Although this is JitServer and not Java, please excluded 8 here as well, just in case. We don't have the machines to add any more special.system testing in the nightly builds.